exchanges/bitstamp/bitstamp.go

```diff
@@ -86,6 +83,23 @@ func (b *Bitstamp) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder)
 	return fee, nil
 }
 
+// GetTradingFee returns a trading fee based on a currency
+func (b *Bitstamp) getTradingFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (float64, error) {
+	var resp TradingFees
+	err := b.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, bitstampAPITradingFees+"/"+strings.ToLower(feeBuilder.Pair.String()), true, nil, &resp)
+	if err != nil {
+		return 0, err
+	}
```

Comment:
Could you please make a separate function for this that returns a slice of TradingFees struct? And allow for an empty currency pair so it returns the entire list of fees for all trading pairs. 

```go
func (b *Bitstamp) GetAccountTradingFees(ctx context.Context, pair currency.Pair) ([]TradingFees, error) {}
```

That way a client can just use this function without having to use the feeBuilder functionality if they chose not too.

Reply:
Let's make it a separate PR as ideally, it should be an interface function to cover all the exchanges to avoid the divergence between the exchanges. What do you think?

Reply:
Just a library function in `bitstamp.go` `GetAccountTradingFees` should suffice, agreed we can make an interface function for it later in a PR.

Reply:
function added